app-crypt/libu2f_host/patches/libu2f_host-1.1.10.patchset

```diff
@@ -0,0 +1,18 @@
+From 2d6f6370722c0f2c41430d2e539d2a18a93f433c Mon Sep 17 00:00:00 2001
+From: Panagiotis Vasilopoulos <hello@alwayslivid.com>
+Date: Fri, 27 Dec 2019 20:10:28 +0200
+Subject: Fixed help2man
+
+
+diff --git a/src/Makefile.am b/src/Makefile.am
+index 5dd747a..777bfbd 100644
+--- a/src/Makefile.am
++++ b/src/Makefile.am
+@@ -40,3 +40,4 @@ u2f-host.1: $(srcdir)/u2f-host.c $(srcdir)/cmdline.ggo $(top_srcdir)/configure.a
+                --output=$@ $(builddir)/u2f-host$(EXEEXT) \
+                --name="Yubico Universal 2nd Factor (U2F) Host Tool" \
+-               --no-info
++               --no-info \
++               --no-discard-stderr
```

Comment:
Patch is OK, you could also just add it after --no-info instead of a new line (nitpicking) :)